enhancements/sig-architecture/63-hosted-addon/README.md

```diff
@@ -0,0 +1,262 @@
+       |                                              |                             |                          
+       |       +--------------------------------------|---------------------+       |                          
+       |       | hosting-cluster                      |connect              |       |                          
+       |       |                                      |                     |       |                          
+       |       |  +---------------------+  +---------------------+          |       |                          
+       |       |  | deployment:         |  | deployment:         |          |       |                          
+  apply|       |  |  registration-agent |  |  hello-addon-agent  |          |       |                          
+       |       |  |  work-agent         |  |                     <-------------------                          
+       |       |  |                     |  |                     |          |                                  
+       |       |  +---------------------+  +---------------------+          |                                  
+       |       |                                      |connect              |                                  
+       |       +--------------------------------------|---------------------+                                  
+       |                                              |                                                        
+       |       +--------------------------------------v---------------------+                                  
+       |       | managed-cluster                                            |                                  
+       |       |                                                            |                                  
+       |       | +---------------------+   +---------------------+          |                                  
+       |       | |CRDs:                |   |Configurations:      |          |                                  
+       |       | | clusterClaim        |   | role,rolebinding,sa |  etc.    |                                  
+       |       | | appliedmanifestwork |   +----------^----------+          |                                  
+       |       | +---------------------+              |                     |                                  
+       |       +--------------------------------------|---------------------+                                  
+       |                                              |                                                        
+       |                                              |                                                        
+       -----------------------------------------------|                                                        
+```
+
+### Guidance for transforming Default mode addon to hosted mode
```

Comment:
This example/guide is very helpful. I think we had done some initial tests with the policy addon using this kind of volumeMount and kubeconfig arg, and it worked reasonably well.